### ISSUE TITLE
non-_id strings inside objects are randomly changed to ObjectIds when handlind document_id's

### ISSUE THREAD

**Issue by Bryteon** [steineey/node-red-contrib-mongodb4#22]
For some reason random strings inside nested objects get changed to ObjectId's in MongoDB Atlas when records are updated.

I use the mongodb4 node to do a updateOne operation. I have the Handle document_id ticked.

In this specific example, I'm adding an array of objects to a record. 

Record Before
![image](https://github.com/steineey/node-red-contrib-mongodb4/assets/116740177/fbe00be2-9bad-4486-b7ba-f2caf6627984)

Payload passed:
![image](https://github.com/steineey/node-red-contrib-mongodb4/assets/116740177/4b8192b8-8a3c-4d7d-a405-fb68cd13e8a2)

Record after
![image](https://github.com/steineey/node-red-contrib-mongodb4/assets/116740177/e5b60a2e-4ce2-4f16-8088-f5df1c18e4bc)

In Application
![image](https://github.com/steineey/node-red-contrib-mongodb4/assets/116740177/cd8617a3-6ffb-4390-9299-45987d857216)

As you can see this happens only to specific strings. It does not seem to be related to the key names but rather the values themselves.  

In this example the mongodb4 node is changing the string: "POD Accepted" to  ObjectID ('504f44204163636570746564')

If I pass any other value such as "POD" or "Issue" using the exact same method, the strings are stored as strings without issue.

It does not appear to happen to strings in the first level of an object but instead only happens to strings inside nested objects.

**Comment by steineey** [comment]
Hi Bryteon,

thank you for reporting this issue. It looks like a bug in the document ID feature of this node.
Can you please send me a example JSON of your payload to reproduce this error.

If "handle document id" is checked, the node searches for potential ObjectId's under the _id key. But in your case it looks like that other strings are also converted to ObjectId.

"handle document id" checked makes only sense if you have a document id string typed and the database is expecting an _id of type ObjectId.

Thank you for your help.

**Comment by steineey** [comment]
Hi Bryteon,

I fixed this issue in version 2.3.0.
```sh
npm install --save --omit=dev node-red-contrib-mongodb4@2.3.0
```

Please reopen this issue if you have still have this bug. Thank you in advance.

**Comment by Bryteon** [comment]
Hi

Sorry for not getting back to you. Thank you so much.

I'll update and do some tests and let you know.
